internal/schema/redact_test.go

```diff
@@ -1,31 +1,24 @@
 package schema
 
-import (
-	"bytes"
-	"testing"
+// func TestRedact(t *testing.T) {
```

Comment:
Currently everything gets stored in the schema, so we're not redacting anything.